src/components/Activity/DefaultTabBar.js

```diff
@@ -1,33 +1,41 @@
-const React = require('react');
-const ReactNative = require('react-native');
-const {
+import React, { Component } from 'react';
+import {
+  Animated,
   Dimensions,
   StyleSheet,
-  Text,
   View,
-} = ReactNative;
-const Button = require('./Button');
+ } from 'react-native';
```

Comment:
7:1 Irregular whitespace not allowed(no-irregular-whitespace)
7:1 Expected linebreaks to be 'LF' but found 'CRLF'(linebreak-style)

These errors are difficult to find but I think the errors may have a potential problem.